snowpack/src/build/build-import-proxy.ts

```diff
@@ -184,23 +184,25 @@ export async function wrapImportProxy({
   config,
 }: {
   url: string;
-  code: string;
+  code: string | Buffer;
```

Comment:
WDYT about replacing all references to "code" with "content"? This should be a bit more consistent since we're no longer only handling code.

Reply:
So I agree, and tried to replace it, and realized that’s part of our plugin API now—a `load()` plugin can provide a `{ code: string, map: string }` object if it’s generating source maps. Even though that’s still an “experimental” API, we should still double-check if we want to make that API change. Spun up an issue here: https://github.com/pikapkg/snowpack/issues/910